content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst

```diff
@@ -0,0 +1,102 @@
+=========
+Packaging
+=========
+
+In Odoo *Inventory*, *packaging* refers to disposable containers holding multiple units of a
+specific product. Each specific packaging must be defined on the individual product form.
+
+For example, different packages for cans of soda, such as a 6-pack, a 12-pack, or a case of 36, need
+to be configured on the individual product form. This is because packagings are product specific,
+and not generic.
+
+Configuration
+=============
+
+To use packagings, navigate to :menuselection:`Inventory app --> Configuration --> Settings`. Then,
+under the :guilabel:`Products` heading, enable the :guilabel:`Product Packagings` feature, and click
+:guilabel:`Save`.
+
+.. image:: packaging/enable-packagings.png
+   :align: center
+   :alt: Enable packagings by selecting "Product Packagings".
+
+Create packaging
+================
+
+Packagings can be created directly on the product form, or from the :guilabel:`Product Packagings`
+page.
+
+From product form
+-----------------
+
+Create packagings on a product form by going to :menuselection:`Inventory app --> Products -->
+Products`, and select the desired product.
+
+Under the :guilabel:`Inventory` tab, scroll down to the :guilabel:`Packaging` section, and click
+:guilabel:`Add a line`. In the table, fill out the following fields:
+
+- :guilabel:`Packaging`: (required) name of packaging that appears on sales/purchase orders as a
+  packaging option for the product.
+- :guilabel:`Contained quantity`: (required) amount of product in the packaging.
+- :guilabel:`Unit of Measure`: (required) measurement unit for quantifying the product.
```

Comment:
```suggestion
- :guilabel:`Packaging` (required): name of packaging that appears on sales/purchase orders as a
  packaging option for the product.
- :guilabel:`Contained quantity` (required): amount of product in the packaging.
- :guilabel:`Unit of Measure` (required): measurement unit for quantifying the product.
```
just feels like this formatting reads a bit cleaner. But, if you disagree, feel free to disregard.